vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -1,9 +1,24 @@
 ---
+  - name: Remove ironic container from source cluster
+    vars:
+      ansible_python_interpreter: /usr/bin/python3
```

Comment:
Is this needed here?

Reply:
this docker_container module is installed using pip3 so that I would explicitly use python3 interpreter. Is it wrong?

Reply:
I think it is not needed here, because we are anyway using python 3 on the source cluster VM.

Reply:
Removed